spec/api/apitest.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const request = require('supertest');
+const app = require('../../server/server');
+
```

Comment:
You need to boot the server in order for the loopback stuff to kick in. Add:

`app.boot();`